src/ProgressOnderwijsUtils/Utils.cs

```diff
@@ -225,9 +225,9 @@ public static int MaandSpan(DateTime d1, DateTime d2)
         /// waarbij nulwaarden voor datum of maand worden omgezet naar de waarde 1
         /// Alleen voor KVA4
         /// </summary>
-        public static DateTime? SLMaybeIncompleteDateConversion([CanBeNull] string incompleteDate)
+        public static DateTime? SLMaybeIncompleteDateConversion(string incompleteDate)

```

Comment:
Raar dat er in de generieke utils een SL-functie staat. Moetenw e deze niet verhuizen naar de Progress.Business assembly?